layouts/main.tsx

```diff
@@ -112,6 +115,14 @@ const Layout = ({ children, title = "Livepeer Explorer" }) => {
     [currentRound?.currentL1Block, pollData]
   );
 
+  const totalActiveProposals = useMemo(() => {
```

Comment:
We can be more explicit here by naming it `totalActiveTreasuryProposals`.

Reply:
I've updated the names `totalActiveProposals` and `proposalsData` to `totalActiveTreasuryProposals` and `treasuryProposalsData`, respectively, for better distinction and clarity.